index.html

```diff
@@ -5,4 +5,39 @@
     <link rel="stylesheet" type="text/css" href="css/reset.css">
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
+  <body>
+    <div>
+      
+    </div>
+    <main>
+    
+    <p>For the fig-swirl: Melt butter over medium heat in a saucepan. Add Brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirrinf frequently, until you have a chunky-jammy mixture. Add salt with one or two stirs, set aside and let cool completely.</p>
```

Comment:
Your indentation is wrong here. Make sure to keep your code correctly lined up for ease of reading/finding bugs.